pay_invoice_stripe.py

```diff
@@ -0,0 +1,75 @@
+import argparse
+import json
+import sys
+from socotratools.client import SocotraClient
+import stripe
+
+# Sample application to pay all invoices from a policyholder using Stripe
+
+# Stripe Test Key
+stripe.api_key = 'sk_test_BQokikJOvBiI2HlWgH4olfQ2'
```

Comment:
Should this be here?

Reply:
Yup - it's their public test key from the developer website

Reply:
Why is there a test key? And how does it tie into making API calls. There's a connection we should explain here